src/libraries/Microsoft.Extensions.Hosting/tests/UnitTests/HostTests.cs

```diff
@@ -55,12 +55,7 @@ public void CreateDefaultBuilder_DoesNotChangeContentRootIfCurrentDirectoryIsWin
         {
             using var _ = RemoteExecutor.Invoke(() =>
             {
-                string systemDirectory = Environment.GetFolderPath(Environment.SpecialFolder.System);
-                if (string.IsNullOrEmpty(systemDirectory))
-                {
-                    // Skip the environments (like Nano Server) where Environment.SpecialFolder.System returns empty - https://github.com/dotnet/runtime/issues/21430
-                    return;
-                }
+                string systemDirectory = Environment.SystemDirectory;
```

Comment:
I assume this doesn't suffer from the same Nano issue?

Reply:
I would expect so. We have tests that verify that Environment.SystemDirectory works on Nano and Environment.SystemDirectory does not.